src/Monolog/Logger.php

```diff
@@ -176,6 +176,25 @@ public function popHandler()
     }
 
     /**
+     * Set handlers, removing all existing ones.
+     * Falsey values will be ignored, and if a map is passed, keys will be ignored.
+     *
+     * @param array $handlers All elements must be of type HandlerInterface
+     * @return $this
+     */
+    public function setHandlers(array $handlers)
+    {
+        $this->handlers = array();
+        foreach($handlers as $handler) {
```

Comment:
Like much of PHP these days, I think Monolog uses something close to PSR2. http://www.php-fig.org/psr/psr-2/

If you set up phpcs with PSR2 it should give you the warnings.